enzyme/tests/Page.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```

Comment:
As @cazfletch mentioned, we should probably make `Page` and `Sidebar` less generic as they're unlikely to be imported/used by other components.